src/core/Form/StatusText/StatusText.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component, ReactNode } from 'react';
+import classnames from 'classnames';
+import { default as styled } from 'styled-components';
+import { baseStyles } from './StatusText.baseStyles';
+import { HtmlSpan, HtmlSpanProps } from '../../../reset';
+import { TokensProp, InternalTokensProp } from 'core/theme';
+import { withSuomifiDefaultProps } from '../../theme/utils';
+
+const baseClassName = 'fi-status-text';
+const statusTextClassNames = {
+  error: `${baseClassName}--error`,
+};
+
+export type InputStatus = 'default' | 'error' | 'success';
+
+interface InternalStatusTextProps extends HtmlSpanProps {
+  /** id */
+  id?: string;
+  /** Chip element content */
+  children?: ReactNode;
+  /** Custom class name for styling and customizing  */
+  className?: string;
+  /** Disable chip */
+  disabled?: boolean;
+  /** Status */
+  status?: InputStatus;
+}
+
+export interface StatusTextProps extends InternalStatusTextProps, TokensProp {}
+
+const StyledStatusText = styled(
+  ({
+    className,
+    children,
+    tokens,
+    status,
+    ...passProps
+  }: StatusTextProps & InternalTokensProp) => (
+    <HtmlSpan
+      {...passProps}
+      className={classnames(className, baseClassName, {
+        [statusTextClassNames.error]: status === 'error',
```

Comment:
Should the success class be added as well, just to account for all the possible values of the InputStatus?

Reply:
I'd leave this for the following issues when this is taken into use for components supporting this. Currently only used for SearchInput and it has a limited API only supporting error and default statuses.

Reply:
That's fine for me for now. It's already good progress that it exists now :)